packages/newspringchurchapp/src/prayer/UserPrayerList/ActionComponent.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import ActionSheet from 'react-native-actionsheet';
+import PropTypes from 'prop-types';
+import { Icon, Touchable } from '@apollosproject/ui-kit';
+
+class ActionComponent extends React.Component {
+  handleShowActionSheet = () => {
+    this.ActionSheet.show();
+  };
+
+  render() {
+    const { options } = this.props;
+    return (
+      <>
```

Comment:
don't need this fragment

Reply:
I do if I want my linter to not yell at me about my JSX needing to have one parent element. Is there another way to do that?